partitioned-heat-conduction/fenics/heat.py

```diff
@@ -22,18 +22,23 @@
   u = u_0             at t = 0
   u = 1 + x^2 + alpha*y^2 + \beta*t
   f = beta - 2 - 2*alpha
+
+The implementation of the higher-order implicit Runge-Kutta methods was mostly taken from:
+    https://github.com/NikoWul/FenicsIrksome
```

Comment:
Good point. I think we should also refer to the corresponding thesis of Nikola and Niklas (in preparation). This clarifies my role as an advisor as well.

```suggestion
For information on the partitioned heat conduction problem using higher-order implicit Runge-Kutta methods see
* "Wullenweber, Nikola. High-order time stepping schemes for solving partial differential equations with FEniCS. Bachelor's thesis at Technical University of Munich, 2021. URL: https://mediatum.ub.tum.de/1621360"
* "Vinnitchenko, Niklas. Evaluation of higher-order coupling schemes with FEniCS-preCICE. Bachelor's thesis at Technical University of Munich, 2023. (in preparation)"

The implementation of the higher-order implicit Runge-Kutta methods is based on: https://github.com/NikoWul/FenicsIrksome
```